FordLabs/retroquest
# Turn off the service worker to prevent app caching

## Overview
Removes the service worker in preparation for the eventual React rewrite merge.
Also should remove issues of people using cached old versions of the application

## Testing Instructions
* Build the application
* Run the application from Spring Boot
* Navigate to the webpage
* See that there is no service worker
